docs/tests/accessibility-tests.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Accessibility Tests
+---
+
+# Accessibility Tests
+
+Accessibility testing is a form of automated and manual testing that helps us
+identify some of the potential accessibility concerns on DEV.
+
+Many a11y issues are not obvious to everyone who contributes to the DEV project,
+therefore leaning on tools to help us identify these issues is a good practice.
+
+It's a good idea to use browser plugins while you're developing to keep an eye
+out for these issues, as well as including automated tests to catch regressions
+in future changes.
+
+The [aXe devtools](https://www.deque.com/axe/extension) is a great place to get
```

Comment:
I'm getting a 404 at that link.

Reply:
u right

Good catch, just fixed it! :sweat_smile: